### DISCUSSION TITLE
Issue with event on destination bucket when using batch replication

### DISCUSSION THREAD

**Discussion by moranmoshe** [minio/minio#18729] [Q&A]
We are using batch replication (push and one way, asynchronous) from source minio to destination minio.
While the replication works fine, we are unable to get notifications using events to KAFKA.
When we tried to upload a file directly to the target bucket, the events fires fine.

Any idea why?

**Comment by harshavardhana** [comment]
Kafka notification must be configured on the bucket that receives it, not the one that is sending data.

**Reply by moranmoshe** [reply]
We did configure the target bucket ("the bucket that receives it").
The events are not fired.
I also mentioned:
"When we tried to upload a file directly to the target bucket, the events fires fine"

Thanks for any help
